lib/core/src/Cardano/Wallet/Primitive/Types/Tx/CBOR.hs

```diff
@@ -0,0 +1,102 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# LANGUAGE ExistentialQuantification #-}
+
+module Cardano.Wallet.Primitive.Types.Tx.CBOR where
```

Comment:
Could you add an explicit export list here?

Reply:
yes